--- target/linux/generic/config-4.14	2020-04-04 11:45:35.311223400 +0800
+++ target/linux/generic/config-4.14	2020-04-04 11:46:07.979502300 +0800
@@ -5812,3 +5812,9 @@
 # CONFIG_ZRAM is not set
 # CONFIG_ZSMALLOC is not set
 # CONFIG_ZX_TDM is not set
+CONFIG_MLX5_FPGA=y
+CONFIG_MLX5_MPFS=y
+CONFIG_MLX5_ESWITCH=y
+CONFIG_MLX5_CORE_IPOIB=y
+CONFIG_SFC_MTD=y
+CONFIG_SFC_MCDI_LOGGING=y
\ No newline at end of file
