--- package/lean/default-settings/files/zzz-default-settings	2020-04-01 21:57:04.957151766 +0800
+++ package/lean/default-settings/files/zzz-default-settings	2020-04-01 21:58:45.536528579 +0800
@@ -29,9 +29,8 @@
 
 ln -sf /sbin/ip /usr/bin/ip
 
-sed -i 's/downloads.openwrt.org/openwrt.proxy.ustclug.org/g' /etc/opkg/distfeeds.conf
-sed -i 's/http:/https:/g' /etc/opkg/distfeeds.conf
-sed -i 's/root::0:0:99999:7:::/root:$1$V4UetPzk$CYXluq4wUazHjmCDBCqXF.:0:0:99999:7:::/g' /etc/shadow
+sed -i 's/downloads.openwrt.org/op.hyird.xyz/g' /etc/opkg/distfeeds.conf
+sed -i 's/snapshots/R20.4.1/g' /etc/opkg/distfeeds.conf
 
 sed -i "s/# //g" /etc/opkg/distfeeds.conf
